app/validators/form_email_validator.rb

```diff
@@ -6,29 +6,16 @@ module FormEmailValidator
 
     before_validation :downcase_and_strip
 
-    validate :email_is_unique
```

Comment:
I'm not seeing any specs that correspond to this change? Like specs that allow a second account to register and confirm an unconfirmed email?

Reply:
Yeah, so this was half dead code that was used by `RegisterUserEmailForm`. I move the logic there to try and make things more clear.

The `RequestPasswordReset` service actually calls the `RegisterUserEmailForm` if we need to create an account for a user instead of sending them a reset email. The changes I made to this code in its new home allow you to create an Account with an email that is unconfirmed on another account with a confirmed email address, if that makes any sense. I can add a test for the case to `RegisterUserEmailForm`.

Reply:
Ok yeah that sounds great! Basically just want to make sure we cover that new logic

Reply:
Good call here. I found a bug where the analytics where reporting the existing confirmed user with the unconfirmed email addresses UUID instead of the new users UUID.